### DISCUSSION TITLE
how to add multiple animation and make it play only onces if its clicked?

### DISCUSSION THREAD

**Discussion by fusignbc** [google/model-viewer#3091] [Q&A]
Is there anyone that could assist me with this?
I want this drawer to open when it is clicked on, and since I have multiple drawers, I want to add the same interstion to all of them. If someone can assist me, it would be greatly appreciated.

Thank you very much.

https://glitch.com/edit/#!/pethub?path=index.html%3A30%3A7

**Comment by elalish** [comment]
Actually @futahei is in the process of adding that feature right now, so it should be available  in  our next version: https://github.com/google/model-viewer/pull/3021
